python/py-distorm/Portfile

```diff
@@ -31,9 +31,12 @@ if {${name} ne ${subport}} {
 
     depends_test-append port:yasm
 
+    pre-test {
+        test.cmd        ${python.bin} python/test_distorm3.py
```

Comment:
`test.env` needs to be in here, not `test.cmd`

Reply:
@reneeotten thanks, I've forgot to make a force push. It should be fixed.